python/hail/methods/tests.py

```diff
@@ -241,6 +244,63 @@ def load_bin(ns, path):
                            load_bin(n_samples, grm_nbin_file),
                            atol=tolerance))
 
+    def test_rrm(self):
+        seed = 0
+        n1 = 100
+        m1 = 200
+        k = 3
+        fst = .9
+
+        dataset = hc._hc1.balding_nichols_model(k,
+                                                n1,
+                                                m1,
+                                                fst=(k * [fst]),
+                                                seed=seed,
+                                                num_partitions=4,
+                                                af_dist=UniformDist(0.1, 0.9)).to_hail2()
+
+        def to_int_python(ds):
+            return BlockMatrix.from_matrix_table(ds['GT'].num_alt_alleles()).to_local_matrix()
+
+        def manual_calculation(ds):
```

Comment:
I have a hard time thinking of digital work as manual labor. :) How about `direct_calculation`?